src/app/features/settings/settings.tsx

```diff
@@ -66,9 +65,9 @@ export function Settings({ triggerButton, toggleSwitchAccount }: SettingsProps)
   return (
     <>
       <DropdownMenu.Root>
-        <DropdownMenu.Trigger>{triggerButton}</DropdownMenu.Trigger>
+        <DropdownMenu.IconButton>{triggerButton}</DropdownMenu.IconButton>
         <DropdownMenu.Portal>
-          <DropdownMenu.Content side="left">
+          <DropdownMenu.Content align="end" side="bottom" sideOffset={8}>
```

Comment:
Thanks @fbwoolf  👍 I had to lookup how to move it to `side=left` as I was asked to move it initially